### DISCUSSION TITLE
How to get Mi-Flora reporting?

### DISCUSSION THREAD

**Discussion by RavenReign** [tony-fav/tasmota-blerry#77] [Q&A]
I have had success with BLErry for Mi temperature sensors (THANK YOU). Now I'm working on adding Mi-Flora sensors. I added the drivers for Xiaomi and updated blerry_config with the mac address of the Mi-Flora, now I can see it in the device list, but there is no data and nothing being sent to MQTT server. 

There are other projects like https://github.com/basnijholt/miflora that seem like they would work, but it would be so nice to have BLErry hand everything.

**Comment by tony-fav** [comment]
Hi. I'm not sure what the solution here is. I can use the Mi Flora fine in my tests (in 1 flower pot). It should run with the Xiaomi driver.

**Comment by cracyfloyd** [comment]
For me its running but i am still missing one sensor "Fertility". How i can add ? can anyone help ?

<img width="348" alt="Bildschirmfoto 2024-09-04 um 15 18 16" src="https://github.com/user-attachments/assets/b768f865-7798-40a1-b0e9-3670adff3cf9">
